pages/reference/base-images/base-images.md

```diff
@@ -19,18 +19,19 @@ excerpt: Docker images maintained by balena
 - Multiple Distributions:
   - [Debian](https://www.debian.org/): buster (10), bullseye (11), bookworm (12) and sid
   - [Alpine](https://alpinelinux.org/): 3.12, 3.13, 3.14, 3.15, 3.16, 3.17 and edge
-  - [Ubuntu](https://www.ubuntu.com/): xenial (16.04), bionic (18.04), focal (20.04), impish (21.10), kinetic (22.10) and jammy (22.04)
-  - [Fedora](https://getfedora.org/): 36, 37 and 38
+  - [Ubuntu](https://www.ubuntu.com/): xenial (16.04), bionic (18.04), focal (20.04), impish (21.10) (EOL), kinetic (22.10) (EOL) , jammy (22.04) and lunar (23.04) (EOL)
+  - [Fedora](https://getfedora.org/): 36 (EOL), 37 and 38
```

Comment:
It seems, 37 and 38 are EOL as well: https://endoflife.date/fedora

Reply:
Didn't mark them as EOL because contacts continues building them even though they are EOL. Do you need to mark them as EOL based on the  https://endoflife.date/fedora and not reflect the contracts?

Reply:
Thanks for mentioning that, we definitely don't want to list versions we aren't building anymore. I have already figured out a way to automate this page, but I have been busy with something. Would be able to take this on next sprint, automate this page and maybe we can add the EOL notices automatically as a stretch feature.

Reply:
@vipulgupta2048 Ok so for all the versions that their contract have been removed (i.e nodejs 16.19.1) https://github.com/balena-io/contracts/pull/421 should i change the docs and remove them completly? Right now my proposed patch doesn't seem to follow what you are describing or should we wait for the automation?

Reply:
Thanks for finding that out. Indeed, let's wait for the automation. I will be able to work on after 2 weeks as our current work cycle ends and we bet on this work since there are a lot of moving parts to get this page automated.

Reply:
Thanks for the clarification. So @vipulgupta2048 will leave this open till the merge of the automation finish just for reference.Feel free to close it when the automation is done or mention me to mark it as closed.